chore: parse diff extensions (#3)
Paperspace/deploy-action: README.md

```

### Config paths

You can supply an optional relative path for your paperspace config like so:

```yaml
- uses: paperspace/deploy-action@main
  name: Deploy action
  id: deploy
  env:
    API_KEY: ${{ secrets.PAPERSPACE_API_KEY }}
  with:
    projectId: p28rlnvnw51
    configPath: ./random/paperspace.jsonc
    image: paperspace/deployment-fixture
```

> By default, the github action will look for a config file in the follow paths:

Reviewer: ```suggestion
By default, the action looks for a config file by order of precedence:
```
